lib/hyrax/configuration.rb

```diff
@@ -267,6 +267,14 @@ def derivative_services
       @derivative_services ||= [Hyrax::FileSetDerivativesService]
     end
 
+    ##
+    # @!attribute [rw] file_set_file_service
+    #   @return [Class] implementer of {Hyrax::FileSetFileService}
+    attr_writer :file_set_file_service
+    def file_set_file_service
+      @file_set_file_service ||= Hyrax::FileSetFileService
+    end
```

Comment:
We have been adding new configurations to `spec/lib/hyrax/configuration_spec.rb` (to test that they are settable and retrievable) and `lib/generators/hyrax/templates/config/initializers/hyrax.rb` (as documentation).

I don’t personally care too much either way but it might be good to follow the pattern?

Reply:
got this in